apm-server.docker.yml

```diff
@@ -198,9 +198,13 @@ apm-server:
 
   #---------------------------- APM Server - Agent Configuration ----------------------------
 
-  # When using APM agent configuration, information fetched from Kibana will be cached in memory for some time.
-  # Specify cache key expiration via this setting. Default is 30 seconds.
-  #agent.config.cache.expiration: 30s
+  #agent:
+    # When using APM agent configuration, information fetched from Kibana will be cached in memory for some time.
+    # Specify cache key expiration via this setting. Default is 30 seconds.
+    #config.cache.expiration: 30s
+
+    #elasticsearch:
+      #api_key: "id:api_key"
```

Comment:
Why is code from the `agentConfig` bleeding into the sourcemapping logic? There is already a `source_mapping.elasticsearch` section - shouldn't that be used?

Reply:
This is what kibana is using: https://github.com/elastic/kibana/pull/147650/files#diff-a94aa3619e34f92c6719ca26e5efa03eb62eb2a077bb98a87cf0c7c9e751e7f8R17

Unless I misunderstood that PR, we need to match the path they are injecting to.

Reply:
Ideally they would inject it at the agentcfg and the sourcemap level. Please check with them how much effort it is to change that.

Reply:
Left a comment on the kibana PR and pinged you for visibility :+1:

Reply:
Changed!